modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/ColumnStatisticsCollector.java

```diff
@@ -62,15 +63,30 @@
     /** Hasher. */
     private final Hasher hash = new Hasher();
 
+    /** Version. */
+    private final long ver;
+
     /**
      * Constructor.
      *
      * @param col Column to collect statistics by.
      * @param comp Column values comparator.
      */
     public ColumnStatisticsCollector(Column col, Comparator<Value> comp) {
+        this(col, comp, 0);
```

Comment:
Again - a method for test only. There are only two lines in ColumnStaisticsCollectorTest to fix to remove it.